recipe/meta.yaml

```
@@ -10,32 +10,48 @@ source:
   sha256: 27eb45ea740ea0d379b3adcd3ba53b538b5176d89daa3e8d96220f15630ae38a
 
 build:
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-
-requirements:
-  host:
-    - python >=3.6
-    - pip
-  run:
-    - python >=3.6
-    # everything here below is in extras_require: 
-    # move to an "optional requirements" field as 
-    # soon as conda allows it
-    - aiofiles
-    - graphene
-    - itsdangerous
-    - jinja2
-    - python-multipart
-    - pyyaml
-    - requests
-    - ujson
+   number: 2
+   noarch: python
 
 test:
   commands:
     - python -c "from starlette import *"
 
+outputs:
+  - name: {{ name }}-core
```

`base` is more common. For eg: `matplotlib-base`
```suggestion
  - name: {{ name }}-base
```